### DISCUSSION TITLE
Allow for Kaggle Embeds

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#20914] [Feature Requests]
@adriens suggested that we add the ability to embed Kaggle Notebook and Datasets!

This came up in discussion in my post on embeds here - https://dev.to/devteam/lesser-known-features-of-dev-embeds-3bpn

Notably, there is an issue that seems semi-related here - https://github.com/forem/forem/issues/18537 but may just be related to the preview link for Kaggle not working.

There are also many other embed-related requests here - https://github.com/forem/forem/discussions?discussions_q=is%3Aopen+embed

**Comment by benhalpern** [comment]
This should be possible.

This is an embed snippet for this link: https://www.kaggle.com/code/abdmental01/optimizing-banana-quality-a-modeling-approach

```
<iframe src="https://www.kaggle.com/embed/abdmental01/optimizing-banana-quality-a-modeling-approach?kernelSessionId=175203292" height="800" style="margin: 0 auto; width: 100%; max-width: 950px;" frameborder="0" scrolling="auto" title="🍌🌟Optimizing Banana Quality A Modeling Approach"></iframe>
```

Taking in `{% embed https://www.kaggle.com/code/abdmental01/optimizing-banana-quality-a-modeling-approach %}` is a matter of translating to the above format (and I assume `kernelSessionId` is optional?)

**Reply by adriens** [reply]
> and I assume kernelSessionId is optional?

Indeed, it is optional. :+1:  It's even better not to provide it :thought_balloon:
